tests/unit/models/transactions/test_nftoken_create_offer.py

```diff
@@ -62,7 +62,7 @@ def test_nftoken_sell_offer_with_positive_amount(self):
             sequence=_SEQUENCE,
             amount="1",
             nftoken_id=_NFTOKEN_ID,
-            flags=[NFTokenCreateOfferFlag.TF_SELL_TOKEN],
+            flags=[NFTokenCreateOfferFlag.TF_SELL_NF_TOKEN],
```

Comment:
I think we're going with `NFTOKEN` instead of `NF_TOKEN` - @khancode added logic to handle that spelling. I think it's more intuitive to treat NFToken as one word.